BeatSaberPlaylistsLib/PlaylistManager.cs

```diff
@@ -164,6 +167,42 @@ public void RequestRefresh(string requestSource)
             PlaylistsRefreshRequested?.Invoke(this, requestSource);
         }
 
+        /// <summary>
+        /// Call this if you want to reload all child managers
+        /// </summary>
+        public void RefreshPlaylists(bool refreshChildren)
+        {
+            IPlaylistHandler handler = DefaultHandler ?? PlaylistHandlers.Values.FirstOrDefault() ?? throw new InvalidOperationException("PlaylistManager has no registered IPlaylistHandlers.");
+
+            if (refreshChildren)
+            {
+                string[] subDirectories = Directory.GetDirectories(PlaylistPath);
+                ChildManagers.Clear();
```

Comment:
Instead of clearing the ChildManagers, can we just add child managers for new directories and update the playlists in the existing child managers?